docs/_articles/en/v3.x/salesforce-graph-engine/rules.md

```diff
@@ -71,13 +71,9 @@ The `stripInaccessible` warning is thrown for all `stripInaccessible` checks on
 
 Graph Engine ran into an error while walking this path. Manually verify that you have a sanitizer on the path, and add an engine directive to skip the path. Next, create a GitHub issue for the Code Analyzer team that includes the error and stack trace so we can research and resolve it. After we determine a fix for the issue, check Code Analyzer [Release Information](./en/v3.x/release-information/) for more info.
 
-#### See Also
+### ApexFlsViolationRule Limitations
 
-- [FAQ](./en/v3.x/faq/#questions-about-interpreting-apexflsviolationrule-results)
-- [Enforce Security With the stripInaccessible Method](https://developer.salesforce.com/docs/atlas.en-us.apexcode.meta/apexcode/apex_classes_with_security_stripInaccessible.htm)
-- [Enforcing Object and Field Permissions](https://developer.salesforce.com/docs/atlas.en-us.apexcode.meta/apexcode/apex_classes_perms_enforcing.htm)
-- [Filter SOQL Queries Using WITH SECURITY_ENFORCED](https://developer.salesforce.com/docs/atlas.en-us.apexcode.meta/apexcode/apex_classes_with_security_enforced.htm)
-- [Frequently Asked Questions](./en/v3.x/faq/)
+- Triggers are currently unsupported.
```

Comment:
Since we mention this in the other Limitations section, and we don't support a few more source types and sinks, can we remove the reference to triggers here?